### PR TITLE
Prune images on target cluster

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1868981

CP for 4.5, 4.6

### CODE REVIEW

**Review by sergiordlr — APPROVED** [review]
Looks good to me.

File: modules/migration-running-migration-plan-cam.adoc [hunk 1]
```diff
@@ -13,13 +13,20 @@ You can stage or migrate applications and data with the migration plan you creat
...
 * Source cluster
-* Target cluster, which is added automatically during the CAM tool installation
+* Target cluster
 * Replication repository
 * Valid migration plan
 
 .Procedure
 
-. Log in to the CAM web console on the target cluster.
+. Log in to the source cluster and delete old images:
```
Comment (kalexand-rh) [inline modules/migration-running-migration-plan-cam.adoc:22]: Consider breaking this step in half:

```
. Log in to the source cluster as a user with ______ permissions.
. Delete old images by using the following command:
```

**Review by kalexand-rh — COMMENTED** [review]
I have a suggestion, but this otherwise LGTM.

File: modules/migration-running-migration-plan-cam.adoc (hunk 1 above)
Comment (apinnick) [reply modules/migration-running-migration-plan-cam.adoc:22]: Permissions step not necessary. Already covered in the entire doc.

Comment (kalexand-rh) [reply modules/migration-running-migration-plan-cam.adoc]: Then I'll merge it when Travis finishes. Thanks!

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Pull request merged**